can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -0,0 +1,151 @@
+                    # ended up in after the split...
+                    # https://en.wikipedia.org/wiki/Cordova,_Alaska
+                    2080: 2261,
+                }
+            )
+        )
+
+        # Set all missing values (-999999) to nan for all numeric columns
+        numeric_cols = list(df.select_dtypes("number"))
+        df.loc[:, numeric_cols] = df.loc[:, numeric_cols].where(lambda x: x > 0, np.nan)
+
+        # Create new columns that we need
+        df["inpatient_beds_used_covid_7_day_avg"] = df.eval(
+            "total_adult_patients_hospitalized_confirmed_covid_7_day_avg + "
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_avg"
+        )
+
+        crename = {
+            "inpatient_beds_7_day_avg": CMU(
+                category="hospital_beds_capacity",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+            "inpatient_beds_used_7_day_avg": CMU(
+                category="hospital_beds_in_use",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+            # This column is generated by summing adult and pediatric
+            # beds -- Should be missing if either is missing
+            "inpatient_beds_used_covid_7_day_avg": CMU(
+                category="hospital_beds_in_use_covid",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+            "total_staffed_adult_icu_beds_7_day_avg": CMU(
+                category="adult_icu_beds_capacity",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+            "staffed_adult_icu_bed_occupancy_7_day_avg": CMU(
+                category="adult_icu_beds_in_use",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+            "staffed_icu_adult_patients_confirmed_covid_7_day_avg": CMU(
+                category="adult_icu_beds_in_use_covid",
+                measurement="rolling_average_7_day",
+                unit="beds",
+            ),
+        }
+
+        # Reshape by putting into long form
+        df_long = df.melt(
+            id_vars=["dt", "location"], value_vars=crename.keys()
+        ).dropna()
+        df_long.loc[:, "value"] = pd.to_numeric(
+            df_long["value"].astype(str).str.replace(",", "")
+        )
+
+        # Add category, measurement, unit, age, sex, race
+        df_long = self.extract_CMU(df_long, crename)
+
+        # Group by relevant factors and sum
+        identifier = [
+            "dt",
+            "location",
+            "category",
+            "measurement",
+            "unit",
+            "age",
+            "sex",
+            "race",
+        ]
+
+        # TODO: We could do a different groupby and put this into states
+        # or hospital regions
+        out_county = df_long.groupby(identifier)["value"].sum().reset_index()
```

Comment:
Is the original data at the "observation" level?

Reply:
The original data has one row for each hospital